src/core_modules/capture-core/components/WidgetEnrollment/Actions/Transfer/hooks/useUpdateOwnership.ts

```diff
@@ -8,31 +6,33 @@ import { errorCreator, FEATURES, useFeature } from 'capture-core-utils';
 import { ProgramAccessLevels } from '../../../TransferModal/hooks/useProgramAccessLevel';
 import { OrgUnitScopes } from '../../../TransferModal/hooks/useTransferValidation';
 
-export type UpdateEnrollmentOwnership = {|
-    orgUnitId: string,
-    programAccessLevel: ?$Values<typeof ProgramAccessLevels>,
+type QueryRefetchFunction = any;
```

Comment:
QueryRefetchFunction is defined in `src/core_modules/capture-core-utils/types/app-runtime.ts`. You should import it and use it and not create a local type